src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -262,14 +262,15 @@ const AnalysisAdd = (
             <Radio
               name="restrictions"
               inline
+              checked={!!form.restrictions.rawValue}
               onClick={updateFormFieldValue(form.restrictions, updateForm, true)}
             >
               Ja
             </Radio>
             <Radio
               name="restrictions"
               inline
-              defaultChecked={!form.restrictions.rawValue}
+              checked={!form.restrictions.rawValue}
```

Comment:
Doesn't this generate a react warning/error related to how we're controlling the value to the component?